### PR TITLE
Sentry.EntityFramework note for require ef update

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
fyi @bruno-garcia

File: src/platforms/dotnet/migration.mdx [hunk 1]
```diff
@@ -6,6 +6,12 @@ sidebar_order: 100
 
 Upgrading includes both breaking changes and new features.
 
+## Migrating to 3.12.0
+
+### NuGet Changes
+
+If using the `Sentry.EntityFramework` NuGet package, an update to at least version 6.2 of the `EntityFramework` NuGet package will also be required.
```
Comment (imatwawana) [inline src/platforms/dotnet/migration.mdx:13]:
```suggestion
If you're using the `Sentry.EntityFramework` NuGet package, you must update to at least version 6.2 of the `EntityFramework` NuGet package.
```

**Review by imatwawana — APPROVED** [review]
Thanks for this! Minor wording changes separate from technical review.

**Review by bruno-garcia — APPROVED** [review]

**Comment by bruno-garcia** [comment]
Thanks @SimonCropp

**Pull request merged**